### PR TITLE
Added Health Dept. Phone number to footer

### PR DESCRIPTION
**Pull request by aravindharinair**
### Issue Reference
Added Kerala Health Dept. 24x7 Helpline Call Centre.

### Summarize
Added 1800 123 1454 to footer.

> Added toll-free number to footer.

@syam3526 @drtechie do verify.

### CODE REVIEW

**Comment by syam3526** [comment]
@IEEEKeralaSection/moderators urgent have a look and merge

**Comment by Neonkraft** [comment]
@syam3526  I've raised #777 which lets the users click on the phone number to open the dial pad.

**Comment by aravindharinair** [comment]
@Neonkraft , that's pretty neat. Can I add that here?

**Comment by Neonkraft** [comment]
@aravindharinair go for it!!

**Comment by aravindharinair** [comment]
@Neonkraft Cheers, mate! Could you re-verify?

File: templates/base.html [hunk 1]
```diff
@@ -80,6 +80,10 @@
     <a href="/disclaimer/" style="font-weight:bold">Disclaimer</a>
     <span>&nbsp;|&nbsp;</span>
     <a href="https://github.com/IEEEKeralaSection/rescuekerala/" target="_blank">GitHub</a>
+    <br /><br />
+    നിങ്ങളുടെ എല്ലാ ആരോഗ്യസംബന്ധ സംശയങ്ങൾക്കും വിളിക്കാം ടോൾ ഫ്രീ നമ്പർ <strong><a href="tel:18001231454">1800 123 1454</a></strong><br />
```
Comment (saba83rish) [inline templates/base.html]: suggestion : to put the entire health info  in a stand alone <div> or <span> for future code maintenance

Comment (aravindharinair) [reply templates/base.html]: Um. I'm sorry, but I didn't get you. Could you clarify?

Comment (aravindharinair) [reply templates/base.html]: How does it look? @saba83rish 
Do re-test and verify.

**Pull request merged**